### DISCUSSION TITLE
Is there any plan to port this wonderful project for CircuitPython

### DISCUSSION THREAD

**Discussion by cjgu00** [n-elia/MAX30102-MicroPython-driver#25] [General]
This project is really attractive and useful to me. However, i must switch the development environment from **MicroPython** to **CircuitPython**. Since the CircuitPython provides plenty of support libraries for peripherals, it makes fast prototyping possible. Certainly, the CircuitPython has no any support for **MAX30102**. Is there any plan to port this wonderful project for CircuitPython? I am a novice and can't do this but eager for that.

**Comment by n-elia** [comment]
Hi, thank you for your kind words.

Unfortunately, I don't have any plans or resources to maintain a fork of this driver to CircuitPython.

Hovever, you can find [this](https://github.com/Haxintosh/MAX30102-CircuitPython-driver) porting for CircuitPhython among the forks. I didn't try it, but it seems to me that the author did a great job, you may give it a try and report your feedback here!

**Comment by cjgu00** [comment]
Thank n-elia!
I had tried the fork for CircuitPython. But it did not work anyway. Maybe there are something tricky.

**Reply by n-elia** [reply]
You can't follow the same instructions to run the examples.  I can't help you with that because I never used CircuitPython. I can only suggest to open an issue on that fork!
